docs/sphinx/ome-tiff/specification.rst

```diff
@@ -366,7 +366,7 @@ example the identifiers could be distributed as follows:
             </UUID>
           </TiffData>
     ...
-          <TiffData FirstC="0" FirstT="1" FirstZ="1" IFD="1" PlaneCount="1">
+          <TiffData FirstC="0" FirstT="1" FirstZ="0" IFD="1" PlaneCount="1">
```

Comment:
Thanks @Tom-TBT for spotting this, it does make more sense to have it showing the first IFD. I have added a suggestion to also update the IFD index as well as the Z index.

```suggestion
          <TiffData FirstC="0" FirstT="1" FirstZ="0" IFD="0" PlaneCount="1">
```

Reply:
Was it acceptable then to have FirstZ="1" with IFD="1"   ?

With this example, I was looking if TiffData was the same between `tubhiswt_C0_TP0.ome.tif` and `tubhiswt_C0_TP1.ome.tif`, but I didn't notice that the IFD was different. 

Sorry for taking on your time for such small details